src/snowflake/cli/api/utils/rendering.py

```diff
@@ -33,66 +34,33 @@ def procedure_from_js_file(env: jinja2.Environment, file_name: str):
     )
 
 
-PROCEDURE_TEMPLATE = dedent(
-    """\
-    CREATE OR REPLACE {{ object_type | upper }} {{ name | upper }}(\
-    {% for arg in signature %}
-    {{ arg['name'] | upper }} {{ arg['type'] }}{{ "," if not loop.last -}}
-    {% endfor %}
-    )
-    RETURNS {{ returns }}
-    LANGUAGE {{ language }}
-    {% if runtime_version is defined -%}
-    RUNTIME_VERSION = '{{ runtime_version }}'
-    {% endif -%}
-    {% if packages is defined -%}
-    PACKAGES = ('{{ packages }}')
-    {% endif -%}
-    {% if imports is defined -%}
-    IMPORTS = ({% for import in imports %}'{{ import }}'{{ ", " if not loop.last }}{% endfor %})
-    {% endif -%}
-    {% if handler is defined -%}
-    HANDLER = '{{ handler }}'
-    {% endif -%}
-    {% if code is defined -%}
-    AS
-    $$
-    {{ code }}
-    $$
-    {%- endif -%}
-    ;
-
-    {%- if grants is defined -%}
-    {%- for grant in grants %}
-    GRANT USAGE ON {{ object_type | upper }} {{ name | upper }}({% for arg in signature %}{{ arg['type'] }}{{ ", " if not loop.last }}{% endfor %})
-    TO DATABASE ROLE {{ grant['role'] }};
-    {% endfor -%}
-    {% endif -%}\
-"""
-)
+_CUSTOM_FILTERS = [read_file_content, procedure_from_js_file]
 
 
-@jinja2.pass_environment  # type: ignore
-def render_metadata(env: jinja2.Environment, file_name: str):
-    metadata = json.loads(
-        SecurePath(file_name).absolute().read_text(file_size_limit_mb=UNLIMITED)
+def _env_bootstrap(env: Environment) -> Environment:
+    for custom_filter in _CUSTOM_FILTERS:
+        env.filters[custom_filter.__name__] = custom_filter
+
+    return env
+
+
+def get_snowflake_cli_jinja_env():
+    _random_block = "___very___unique___block___to___disable___logic___blocks___"
```

Comment:
What is it for?

Reply:
A unique block to disable logic blocks. We don't want to support `{% for foo in bar %}` syntax.